sites/healthcarepackaging/server/templates/dynamic-page/index.marko

```diff
@@ -8,10 +8,9 @@ $ const { id, alias, pageNode } = data;
       <marko-web-gtm-push data=context />
     </marko-web-gtm-dynamic-page-context>
   </@head>
-  <@above-container>
-  </@above-container>
   <@page>
-    <marko-web-gam-define-display-ad ...GAM.getAdUnit({ name: "lb1" }) modifiers=["top-of-page"] />
+    <marko-web-gam-define-display-ad ...GAM.getAdUnit({ name: "leaderboard" }) modifiers=["top-of-page"] />
```

Comment:
For future reference -- this should be loaded at the top of the page using slots, then loaded via `<marko-web-gam-display-ad>`.
Otherwise, these have to make individual request for each ad unit, and roadblocking will not longer be supported.

Not necessary to change with this PR, but _all_ templates should be reviewed and fixed.